backend/src/Edna/DB/Connection.hs

```diff
@@ -36,11 +38,23 @@ createConnPool
   -> m ConnPool
 createConnPool (ConnString connStr) maxConnsNum = liftIO $ ConnPool <$>
   createPool
-  (connectPostgreSQL connStr)   -- connection creation action
-  close                         -- connection destroy action
-  1                             -- number of individual pools (just one is fine)
-  nominalDay                    -- maximum time the connection should remain open while unused
-  maxConnsNum                   -- maximum number of DB connections.
+  (withRetry 8 $ connectPostgreSQL connStr)  -- connection creation action
+  close        -- connection destroy action
+  1            -- number of individual pools (just one is fine)
+  nominalDay   -- maximum time the connection should remain open while unused
+  maxConnsNum  -- maximum number of DB connections.
+  where
+    aSecond = 1e6
+    -- We try to reconnect in case of an error because if we launch postgres
+    -- and edna-server at nearly the same time, edna-server may try to connect
+    -- before postgres starts accepting connections.
+    -- It may happen if you deploy using @docker-compose@ for example.
```

Comment:
As I can see we use `depends_on` in docker-compose, so backend service going to be run after postgres one.
Anyway I think it's good enough to have such a mechanism for DB connection

Reply:
`depends_on` ensures that backend container does not start before postgres one, but we still cannot be sure that postgres has finished initializing and is ready to accept connections when backend tries to connect to it, it may take some time for postgres to initialize.

Reply:
@zhenyavinogradov thanks for information, I'll remember it